.dotnet.azure/src/Custom/FineTuning/AzureCreateJobOperation.Protocol.cs

```diff
@@ -0,0 +1,95 @@
+﻿using System.ClientModel;
+using System.ClientModel.Primitives;
+using System.Diagnostics.CodeAnalysis;
+
+#nullable enable
+
+namespace Azure.AI.OpenAI.FineTuning;
+
+/// <summary>
+/// A long-running operation for creating a new model from a given dataset.
+/// </summary>
+public class AzureCreateJobOperation : CreateJobOperation
+{
+    private readonly PipelineMessageClassifier DeleteJobClassifier = PipelineMessageClassifier.Create(stackalloc ushort[] { 204 });
+    private readonly ClientPipeline _pipeline;
+    private readonly Uri _endpoint;
+    private readonly string _jobId;
+
+    private readonly string _apiVersion;
+
+    internal AzureCreateJobOperation(
+        ClientPipeline pipeline,
+        Uri endpoint,
+        string jobId,
+        string status,
+        PipelineResponse response,
+        string apiVersion)
+        : base(pipeline, endpoint, jobId, status, response)
+    {
+        _pipeline = pipeline;
+        _endpoint = endpoint;
+        _jobId = jobId;
+        _apiVersion = apiVersion;
+    }
+
+    [Experimental("AOAI001")]
+    public virtual ClientResult DeleteJob(string fineTuningJobId, RequestOptions? options)

```

Comment:
Should these override the methods on the base type?  If not, help me understand what's different here.